src/main/java/frc/robot/Ports.java

```diff
@@ -1,13 +1,11 @@
 package frc.robot;
 
 public final class Ports {
-    public static class ExampleSubsystem {
-        public static final int MAIN = 0;
-        public static final int AUX = 0;
-        public static final boolean IS_MAIN_INVERTED = false;
-        public static final boolean IS_AUX_INVERTED = false;
-        public static final boolean IS_MAIN_SENSOR_INVERTED = false;
-        public static final boolean IS_AUX_SENSOR_INVERTED = false;
+    public static class Shooter {
+        public static final int SOLENOID = 0;
+        public static final int MAIN_MOTOR = 0;
+        public static final boolean MAIN_INVERTED = false;
+        public static final boolean MAIN_SENSOR_PHASE = false;
```

Comment:
Rename to `IS_ENCODER_INVERTED` or `IS_ENCODER_IN_PHASE`. First, because from the name it's not clear what `false` means. Second, there are many types of sensors, names need to be specific and unambiguous.